types/react-native-material-menu/react-native-material-menu-tests.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { View, Text } from 'react-native';
+import Menu, { MenuItem, MenuDivider } from 'react-native-material-menu';
+
+class App extends React.PureComponent {
+    _menu: Menu | null = null;
+
+    setMenuRef = (ref: Menu) => {
+        this._menu = ref;
+    };
+
+    hideMenu = () => {
+        this._menu?.hide();
+    };
+
+    showMenu = () => {
+        this._menu?.show();
+    };
```

Comment:
if we change those to:
```ts
    setMenuRef = (ref: Menu) => (this._menu = ref);

    hideMenu = () => this._menu && this._menu.hide();

    showMenu = () => this._menu && this._menu.show();
```
this would fix couple of linting errors (check other comment). The elvis notation requires TS 3.7, but you probably don't want to limit usage of the types by setting this requirement high.